packages/react/time-series/tsconfig.json

```diff
@@ -1,19 +1,9 @@
 {
+  "extends": "../../../tsconfig.json",
   "compilerOptions": {
-    "baseUrl": ".",
-    "jsx": "preserve",
-    "target": "es2016",
-    "moduleResolution": "node",
-    "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "strict": true,
-    "skipLibCheck": true,
-    "paths": {
-      "@/testing/*": ["testing/*"],
-      "@/testing": ["testing"],
-      "@/time-series": ["src"]
-    }
```

Comment:
If I reintroduce this, maybe I don't need to exclude `__test__`? Then again, if I include `__test__`, we need to put `testing` in the `rootDir` of each package. It's something else to change…

Reply:
Would rather address this later.